### DISCUSSION TITLE
Decorators and Web components

### DISCUSSION THREAD

**Discussion by hellagutmann-soelen** [storybookjs/storybook#26029] [Help]
### Summary

Hey, we need to wrap our web components around a web component which is responsible for theming.

As far as we can see decorators do not work for webcomponents:

https://storybook.js.org/docs/writing-stories/decorators

Is there a way around in order to wrap around `#storybook-preview-wrapper` and `#storybook-docs`?

Thanks in advance

**Comment by break-stuff** [comment]
Web components should work fine with decorators. What issue were you running into?
